service/controller/v18/cloudconfig/cloud_config_test.go

```diff
@@ -1,16 +1,16 @@
 package cloudconfig
 
 import (
-	"bytes"
-	"compress/gzip"
 	"context"
 	"encoding/base64"
-	"io"
 	"strings"
 	"testing"
 
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	ignition "github.com/giantswarm/k8scloudconfig/ignition/v_2_2_0"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_4_0_0"
```

Comment:
Do we really need the new packages here? Just wondering. It adds complexity of updating Cloud Configs and Ignition more complex. Also, why are the versions different? Why `2.2.0` and `4.0.0`?

Reply:
Because `ignition` package is related only to ignition and it is useless to copy it with cloudconfig everytime. Also, `2.2.0` is current spec of the ignition.